### DISCUSSION TITLE
How to format a samba share for a url in _settings.yml

### DISCUSSION THREAD

**Discussion by abeaudin** [dadoonet/fscrawler#1768] [Q&A]
I'm trying to use fscrawler for workplace search and would like the URL for the documents indexed to be \\servername\directory but I'm not having much luck getting the proper format for the share in the url field.

I've tried:

```
smbclient -L \\vertige -I 192.168.0.25 -N

	Sharename       Type      Comment
	---------       ----      -------
<snip>
	sambashare      Disk      Samba on Vertige
</snip>
```

and in the _settings file

```
name: "WOT"
fs:
  url: "\\\\vertige\\sambashare"
```
but I get

```
13:53:04,199 INFO  [f.p.e.c.f.FsParserAbstract] FS crawler started for [WOT] for [\\vertige\sambashare] every [1m]
13:53:04,202 WARN  [f.p.e.c.f.FsParserAbstract] Error while crawling \\vertige\sambashare: \\vertige\sambashare doesn't exists.
13:53:04,202 INFO  [f.p.e.c.f.FsParserAbstract] FS crawler is stopping after 1 run
```
similar for //vertige/sambashare

two questions
1. Is this URL format supported?
2. If it's supported, how would I specify the credentials for the samba share or alternatively specify that it's guest/anonymous access?
Thanks/Merci

**Comment by dadoonet** [comment]
Hi @abeaudin and thank you for trying FSCrawler ;) 

Sorry for the delay.

> Is this URL format supported?

Could you try with:

```yml
name: "WOT"
fs:
  url: "//vertige/sambashare"
```

I guess it's mounted on a Windows machine, right? Could you try to assign a letter to the drive? And then try with something like:

```yml
name: "WOT"
fs:
  url: "z:/sambashare"
```

or

```yml
name: "WOT"
fs:
  url: "z:\\sambashare"
```

See https://fscrawler.readthedocs.io/en/latest/admin/fs/local-fs.html#root-directory

> If it's supported, how would I specify the credentials for the samba share

FSCrawler supposes that you have a read access to the dir. There's no auth integration for this input part.

> alternatively specify that it's guest/anonymous access?

I guess you need to do this while "mounting" the samba dir on your system. I mean that FSCrawler does not read "samba" but a drive which is attached to the system.
